### ISSUE TITLE
stdout read error 

### ISSUE THREAD

**Issue by nitimohi** [COUR4G3/python-pppd#12]
i ma getting the error:
File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.9/dist-packages/pppd.py", line 82, in __init__
    self.output += self.proc.stdout.read()
TypeError: can't concat NoneType to bytes

i started chat-script with -e option so the process starts returning stdout ass soon as the process is started, not i still get this error as my process return anything on stdout after at least  1 sec.
is it batter to give some delay before read or or check the type of return ?

**Comment by nitimohi** [comment]
i am using it raspberry pi, with SIM7000G

**Comment by nitimohi** [comment]
Latest code in repo don't have this issue, nut pip didn't pull the latest code

**Comment by david-kooi-dsi** [comment]
Same issue on an RPI4.

**Comment by david-kooi-dsi** [comment]
The issue occurred in a pip installed python-pppd. I pulled latest master and that version works for me.

**Comment by COUR4G3** [comment]
Yeah, I realized the pip release and source are not inline and I've been meaning to fix that, I'm gonna do a new release in the morning.
________________________________
